### ISSUE TITLE
need more details for readme

### ISSUE THREAD

**Issue by snowden-fu** [tuna/oh-my-tuna#8]
I have ran it and just got the notice:
`[pypi]: Activating...
[pypi]: Mirror has been activated`
what does it mean? is it ok to work?

**Comment by alick** [comment]
Yeah. Why not? It says pypi mirror has been activated. Maybe it's better to say "The TUNA mirror of pypi has been activated. You are good to go!"?
